lesson16/main.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+const jsonUrl = "https://jsondata.okiba.me/v1/json/BFwf7210926001715";
+const newsWrapper = document.getElementById("js-news");
+const tabUl = document.querySelector(".news__tabs");
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+//comment-icon
+const commentIcon = createElementWithClassName("span", "comment-icon");
+
+//new-label
+const newLabel = createElementWithClassName("span", "new-label");
+newLabel.textContent = "NEW!";
+
+//news-img
+const imgWrapper = createElementWithClassName("div", "news__img");
+
+//tab
+const createNewTab = (data) => {
+  const tabUl = document.querySelector(".news__tabs");
```

Comment:
IMO
i think this class name is lead to mistake. because, To change class name often happen when you write css. So you better to write `js-news-tabs`

Reply:
I understand. I use `.news__tabs` for CSS style, so I implemented it as new id. 
And also I deleted line25 because I already declared `tabUl` on line5(it's global).